libgstc/pygstc/gstc.py

```diff
@@ -906,6 +906,30 @@ def pipeline_get_graph(self, pipe_name):
         result = self._send_cmd_line(['pipeline_get_graph'] + parameters)
         return result
 
+    def pipeline_verbose(self, pipe_name, value):
+        """
+        Set the pipeline verbose mode.
+
+        Parameters
+        ----------
+        pipe_name: string
+            The name of the pipeline
+        value: boolean
+            True or False to activate or deactivate
+ 
+        Raises
+        ------
+        GstdError
+            Error is triggered when Gstd IPC fails
```

Comment:
This is not true. A gstd error is caused when there is an error in the server, not because of an IPC error. The descriptions are inverted.